backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -16,6 +19,30 @@
 
 
 class BibtexWriter(Writer):
+    def _encode(self, text):
+        import latexcodec  # NOQA
+
+        if "$" not in text:
+            return codecs.encode(text, "ulatex+{}".format(self.encoding))
+        else:
+            math_expression_regex = re.compile(r"(?<!\\)\$.*?\$")
```

Comment:
it's better to put this kind of constants out of function bodies, so you don't need to evaluate it on every function call and it's more visible. Here you can put it on the class.